web-marketplace/package.json

```diff
@@ -90,7 +92,8 @@
     "graphql:codegen": "graphql-codegen --config codegen.yml",
     "graphql:codegen-sanity": "graphql-codegen --config sanity-codegen.yml",
     "graphql:codegen-indexer": "graphql-codegen --config indexer-codegen.yml",
-    "analyze": "source-map-explorer 'build/static/js/*.js'"
+    "analyze": "source-map-explorer 'build/static/js/*.js'",
+    "messages:extract": "lingui extract"
```

Comment:
should we integrate `lingui compile` as part of our build process?

Reply:
The compile step is already handled by the Vite plugin.

Reply:
ok nice!